managed/models/database.go

```diff
@@ -727,6 +727,11 @@ var databaseSchema = [][]string{
 			PRIMARY KEY (id)
 		)`,
 	},
+	66: {
+		`UPDATE settings SET settings = settings #- '{ia, enabled}';`,
+		`UPDATE settings SET settings = settings - 'ia' || jsonb_build_object('alerting', settings->'ia');`,
+		`UPDATE ia_rules SET disabled = TRUE`,
```

Comment:
why do we disable ia rules?

Reply:
Because it's old rules from IA implementation. We don't have UI for them anymore, user will not able to remove or disable them anymore. So we disable them automatically and we plan to release some script that will migrate this rules to new alerting system (partially).

In future releases we will drom IA related tables completely.